src/commands.ts

```diff
@@ -656,6 +657,25 @@ function restartLSPServer(workspaceContext: WorkspaceContext): Promise<void> {
     return workspaceContext.languageClientManager.restart();
 }
 
+/** Request that the SourceKit-LSP server reindexes the workspace */
+function reindexProject(workspaceContext: WorkspaceContext): Promise<unknown> {
+    return workspaceContext.languageClientManager.useLanguageClient(async (client, token) => {
+        try {
+            return await client.sendRequest(reindexProjectRequest, {}, token);
+        } catch (err) {
+            const error = err as { code: number; message: string };
+            // methodNotFound, version of sourcekit-lsp is likely too old.
+            if (error.code === -32601) {
+                vscode.window.showWarningMessage(
```

Comment:
Is there a way to check this in advance so that we can only show the command in the command palette if background indexing is enabled?

Reply:
SourceKit-LSP should report this capability